src/Params/DisallowedCallParamValueExcept.php

```diff
@@ -14,9 +15,14 @@ class DisallowedCallParamValueExcept extends DisallowedCallParamValue
 
 	public function matches(Type $type): bool
 	{
+		if ($type instanceof ConstantArrayType) {
+			return true;
```

Comment:
This unfortunately allow all array params even if they should be disallowed. I'll commit a test that fails when it shouldn't.

Reply:
ah fair, it didn't seem as though the configuration allowed for constant arrays to be specified due to the class generics, so this seemed like a safe shortcut, but you are correct it's too simple of a check if it can be

Reply:
You're right, I just haven't thought about arrays until I've seen your PR :-)

Reply:
Arrays in param configs are now officially unsupported and an exception is thrown when used, starting with #177.